core/templates/dev/head/pages/exploration-editor-page/services/exploration-property.service.ts

```diff
@@ -89,10 +89,6 @@ angular.module('oppia').factory('ExplorationPropertyService', [
           return;
         }
 
-        if (angular.equals(this.displayed, this.savedMemento)) {
```

Comment:
Why do we need to delete this?

Reply:
I've removed this condition because it is useless, according to the condition above.

```
if (!this._isValid(this.displayed) || !this.hasChanged()) {
          this.restoreFromMemento();
          return;
}
```

Let's supposed I didn't remove this condition.

If `isValid` is false, then `restoreFromMemento` is called (this function will copy `savedMemento` into `displayed`), then it will return.

If `isValid` is true and `hasChanged` is false, the case will be the same.

But, if `isValid` is true and `hasChanged` is true either, then the deleted condition would be false. The deleted condition would never be true, so it's useless. Do you agree with me? Did you think any other situation that this condition may be important?